flow360/component/simulation/outputs/outputs.py

```diff
@@ -168,13 +178,19 @@ class SurfaceProbeOutput(Flow360BaseModel):
     """
 
     name: str = pd.Field()
-    entities: EntityList[Point] = pd.Field(None, alias="probe_points")
+    entities: EntityList[Point, PointArray] = pd.Field(None, alias="probe_points")
     # Maybe add preprocess for this and by default add all Surfaces?
     target_surfaces: EntityList[Surface] = pd.Field()
 
     output_fields: UniqueItemList[SurfaceFieldNames] = pd.Field()
     output_type: Literal["SurfaceProbeOutput"] = pd.Field("SurfaceProbeOutput", frozen=True)
 
+    @pd.field_validator("entities", mode="after")
+    @classmethod
+    def check_entities_type(cls, value):
```

Comment:
same here. Let's try making the name as explanatory as possible.

Reply:
Updated